### ISSUE TITLE
[NOTICE] About Makefile

### ISSUE THREAD

**Issue by seongil-wi** [KAIST-IS521/TeamThree#22]
각자 폴더의 Makefile은 root에 있는 Top-level Makefile에 의해 building됩니다. 이때 다음과 같은 rule을 정하면 좋을 것 같습니다.
- 생성되는 executable은 Top-level Makefile에서의 make명령어에 의해 만들어지는 "build"폴더에 모두 저장하도록 합시다! 각자 Makefile에서의 executable path를 ../build로 잡으시면 됩니다.
- Build target은 다음과 같은 방법으로 추가합니다. DNSChecker옆에 target directory name을 추가해 주시면 됩니다.
```Makefile
subdir = DNSChecker          #The name of the subdirectory you want to build
```

감사합니다!

**Comment by pr0v3rbs** [comment]
제가 flask를 이용해서 web application을 만들고 있는데요. Makefile 안에다가 flask 다운로드 하는 명령어도 넣는게 좋겠죠?

**Comment by seongil-wi** [comment]
네! 자신의 directory이름을 top-level Makefile에 명시하시고 그 directory에 있는 Makefile에서 download 명령어 넣으시면 됩니다.(혹시 다른 build는 하지않고 install만 하실예정이면 top-level에 명령어를 추가하셔도 좋을 것 같습니다)
